container.go

```diff
@@ -101,28 +102,29 @@ type ContainerRequest struct {
-	NetworkAliases  map[string][]string // for specifying network aliases
-	NetworkMode     container.NetworkMode
-	Resources       container.Resources
-	Files           []ContainerFile // files which will be copied when container starts
-	User            string          // for specifying uid:gid
-	SkipReaper      bool            // indicates whether we skip setting up a reaper for this
-	ReaperImage     string          // alternative reaper image
-	AutoRemove      bool            // if set to true, the container will be removed from the host when stopped
-	AlwaysPullImage bool            // Always pull image
-	ImagePlatform   string          // ImagePlatform describes the platform which the image runs on.
-	Binds           []string
-	ShmSize         int64    // Amount of memory shared with the host (in bytes)
-	CapAdd          []string // Add Linux capabilities
-	CapDrop         []string // Drop Linux capabilities
+	ExtraHosts      []string                                                          // Deprecated: Use PreCreationHook instead
+	Privileged      bool                                                              // Deprecated: Use PreCreationHook instead. For starting privileged container
+	Networks        []string                                                          // for specifying network names
+	NetworkAliases  map[string][]string                                               // for specifying network aliases
+	NetworkMode     container.NetworkMode                                             // Deprecated: Use PreCreationHook instead
+	Resources       container.Resources                                               // Deprecated: Use PreCreationHook instead
+	Files           []ContainerFile                                                   // files which will be copied when container starts
+	User            string                                                            // for specifying uid:gid
+	SkipReaper      bool                                                              // indicates whether we skip setting up a reaper for this
+	ReaperImage     string                                                            // alternative reaper image
+	AutoRemove      bool                                                              // Deprecated: Use PreCreationHook instead. If set to true, the container will be removed from the host when stopped
+	AlwaysPullImage bool                                                              // Always pull image
+	ImagePlatform   string                                                            // ImagePlatform describes the platform which the image runs on.
+	Binds           []string                                                          // Deprecated: Use PreCreationHook instead
+	ShmSize         int64                                                             // Deprecated: Use PreCreationHook instead. Amount of memory shared with the host (in bytes)
```

Comment:
We have this in 1st level, since required for Browser images.

Reply:
let me update the description with all the fields that were pushed from the hostConfig (level 2) to the 1st level